packages/reactivated/generator.tsx

```diff
@@ -20,84 +20,79 @@ const {urls, templates, types, values} = schema;
 
 const project = new Project();
 
-/*
-const urls = {
-    'widget_list': {'widget_id': 'number', 'category_id': 'string'},
-    'create_widget': {},
-    'widget_detail': {'pk': 'number'},
-};
-*/
 const interfaces = project.createSourceFile("");
 
-interfaces.addVariableStatement({
-    declarationKind: VariableDeclarationKind.Const,
-    declarations: [
-        {
-            name: "urls",
-            initializer: JSON.stringify(urls),
-        },
-    ],
-});
-
-const urlMap = interfaces.addInterface({
-    name: "URLMap",
-});
-urlMap.setIsExported(true);
+if (Object.keys(urls).length !== 0) {
+    interfaces.addVariableStatement({
+        declarationKind: VariableDeclarationKind.Const,
+        declarations: [
+            {
+                name: "urls",
+                initializer: JSON.stringify(urls),
+            },
+        ],
+    });
 
-const withArguments = [];
-const withoutArguments = [];
+    const urlMap = interfaces.addInterface({
+        name: "URLMap",
+    });
+    urlMap.setIsExported(true);
 
-for (const name of Object.keys(urls)) {
-    const properties = urls[name as keyof typeof urls].args;
-    const normalizedName = name.replace(/[^\w]/g, "_");
+    const withArguments = [""];
+    const withoutArguments = [""];
```

Comment:
This needs to be handled properly. Cases could be no urls with args, no urls without args, no urls at all.